pylint/checkers/variables.py

```diff
@@ -1917,6 +1965,7 @@ def _check_unused_arguments(self, name, node, stmt, argnames):
 
         self.add_message("unused-argument", args=name, node=stmt, confidence=confidence)
 
+    @utils.check_messages("cell-var-from-loop")
```

Comment:
```suggestion
```
Not a `visit_xxx` method.

Btw. there are a few other `visit_xxx` method in this class that don't have the `check_messages` decorator. If they modify `self._to_consume`, I would say we should leave them alone. (`visit_classdef`, `leave_classdef`, `visit_lambda`, `visit_functiondef`, ...)

Reply:
Do we want to remove those now? I'm looking at `visit_import` for example

Reply:
`visit_import` is fine. Unless I'm missing something? Usually it does make sense to add the decorator, this class is somewhat special though with the `NamesConsumer`.

Reply:
Ah, yeah I was looking at the wrong sub-method. Sorry for the confusion!